api4/post.go

```diff
@@ -809,6 +809,11 @@ func updatePost(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	post.Id = c.Params.PostId
 
+	if *c.App.Config().ServiceSettings.PostEditTimeLimit != -1 && model.GetMillis() > originalPost.CreateAt+int64(*c.App.Config().ServiceSettings.PostEditTimeLimit*1000) && post.Message != originalPost.Message {
```

Comment:
We'll want to allow OAuth app tokens to bypass this check based on discussion here https://community.mattermost.com/private-core/pl/ogxrg9ytg3rxtemwo1jao5jwec

Reply:
@mickmister Would you be ok with opening a new PR for that and assigning it to Integrations? I'm not sure of the requirements for that. Iiuc, this PR will not change anything for OAuth apps, correct?

Reply:
I'm going to merge now to avoid issues caused by monorepo migration. Thanks!